api/envoy/extensions/filters/listener/tls_inspector/v3/tls_inspector.proto

```diff
@@ -17,4 +17,8 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 message TlsInspector {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.config.filter.listener.tls_inspector.v2.TlsInspector";
+
+  // Populate connection fingerprint using data from the SSL Client Hello packet
+  // Defaults to false.
```

Comment:
No need to state a default for scalar types in proto3.

Is the reason to add an API field due to the CPU overhead of computing this if not needed?